tests/debug/test_xdd_debug_init.sh

```diff
@@ -7,10 +7,13 @@
 # Description - Just writes out to /dev/null using XDD with -debug INIT and verifies
 #               that the NUMA cpus are listed if requested through the debug flag
 #
+# Get absolute path to script
+SCRIPT=$(realpath "$0")
```

Comment:
`${BASH_SOURCE[0]}`

https://stackoverflow.com/questions/35006457/choosing-between-0-and-bash-source